frontend/src/metabase/admin/datamodel/containers/FieldApp.jsx

```diff
@@ -297,6 +300,32 @@ const FieldGeneralPane = ({
       />
     </Section>
 
+    {isa(field.semantic_type, TYPE.SerializedJSON) &&
```

Comment:
This isn't great because a user can change the semantic_type of a field. But we can only unfold JSON for fields with the correct database types.

https://github.com/metabase/metabase/pull/28849 should fix it.